internal/store/hpa.go

```diff
@@ -223,6 +224,9 @@ var (
 					Metrics: ms,
 				}
 			}),
+			AggregateBy: map[string]aggregation.Aggregation{
+				"namespace": aggregation.ByLabels("namespace", "condition", "status"),
```

Comment:
I'm a little confused here as to what it means to add all three -- "namespace", "condition", "status" -- does that generate three separate aggregates? In the .md file above you only call out kube_hpa_status_condition_sum_by_namespace.